build.gradle.kts

```diff
@@ -115,6 +115,23 @@ subprojects {
     test {
       useJUnitPlatform()
 
+      exclude("**/AnchorPlatformCustodyEnd2EndTest**")
+      exclude("**/AnchorPlatformCustodyApiRpcEnd2EndTest**")
```

Comment:
Why are these tests excluded?

Reply:
There was a requirement:

`
We would like to see a E2E tests with the Fireblocks.
This test should include all currently implemented features (such as generating deposit address, making a payment, receiving user transfer)
The test should consume API keys for Fireblocks from the env variables (you may want to use TestRunner's profiles for that)
Stellar team would use their own keys to test it on their side.
Importantly, this test should not be part of a regular build-flow. As such, it's recommended to create a separate sourceSet (testFireblocksE2E ) that is not part of a regular flow.
Stellar team will test the functionality with running command ./gradlew testFireblocksE2E  with their Fireblocks API key.
`

Since those tests make real calls to Fireblocks it's necessary to specify Fireblocks Vault Account Id, Api Key and Secret Key.
Custody e2e tests can be executed with a following command:
`CUSTODY_TYPE=fireblocks CUSTODY_FIREBLOCKS_VAULT_ACCOUNT_ID=<your_value> SECRET_CUSTODY_FIREBLOCKS_API_KEY=<your_value> SECRET_CUSTODY_FIREBLOCKS_SECRET_KEY=<your_value> ./gradlew clean testFireblocksE2E`